conf.py

```diff
@@ -217,14 +217,16 @@
         "https://github.com/"
         + html_context["github_user"]
         + "/panda_moveit_config/blob/"
-        + "melodic-devel"
+        + "rolling-devel"
         + "/%s",
         "",
     ),
     "moveit_resources_codedir": (
         "https://github.com/"
-        + html_context["github_user"]
-        + "/moveit_resources/tree/ros2"
+#        + html_context["github_user"]
+        + "130s"
```

Comment:
Reminder to revert the changes in this line and 2 lines below, once https://github.com/ros-planning/moveit_resources/pull/181 is merged.